examples/hybrid/hyperdiffusion.jl

```diff
@@ -37,9 +37,12 @@ function hyperdiffusion_tendency_clima!(Yₜ, Y, p, t)
         ᶜρ = Y.c.ρ
         ᶜuₕ = Y.c.uₕ
         (; ᶜp, ᶜχ, ᶜχuₕ) = p # assume ᶜp has been updated
-        (; ghost_buffer, κ₄, divergence_damping_factor, use_tempest_mode) = p
+        (; ghost_buffer, κ₄, divergence_damping_factor, use_tempest_mode, disable_qt_hyperdiffusion) = p
         point_type = eltype(Fields.local_geometry_field(axes(Y.c)).coordinates)
         is_ρq_tot = :ρq_tot in propertynames(Y.c)
+        if is_ρq_tot && disable_qt_hyperdiffusion
+            is_ρq_tot = false # TODO: reconcile this with ρe_tot or remove if instability fixed with limiters
+        end
```

Comment:
```suggestion
        # TODO: reconcile this with ρe_tot or remove if instability fixed with limiters
        is_ρq_tot = :ρq_tot in propertynames(Y.c) && !disable_qt_hyperdiffusion
```

Is this maybe clearer? Better to define variables once, I think.